test/k8sT/manifests/hubble-cli.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hubble-cli
+spec:
```

Comment:
The namespace does not seem to be defined, is this on purpose?

Reply:
Right now it's deployed in the same namespace as cilium-agent, which tends to be either `cilium` or `kube-system`. I thought it less confusing to omit it therefore. But I can change that.

Reply:
I tried it out: The namespace must not be present, otherwise we cannot deploy this DaemonSet on GKE.